services/dy-static-file-server/docker/custom/Dockerfile

```diff
@@ -7,13 +7,21 @@ FROM nginx:1.21.0-alpine as production
 #     docker run dy-static-file-server:prod
 #
 
+# making nginx user simialar => # simcore-user uid=8004(scu) gid=8004(scu) groups=8004(scu)
+# shadow required for groupmod and usermod
+RUN apk --no-cache add shadow && \
+    groupmod -g 8004 nginx && \
+    usermod -u 8004 -g 8004 nginx
+
```

Comment:
ok so as discussed a few minutes ago. this approach has the downside that this is then written in the dockerfile.
it's faster than doing it in the script, but if these happen to change. or if you run it in your local machine, where you probably do not have the 8004 user you're doomed.

The right way is currently to go:
```bash
make new-service
# follow the cookiecutter instructions, select an alpine-based one to be compatible with your service
# copy the generated entrypoint.sh into your service and use it.
```

Then for later, I would recommend to understand what these guys are doing:
[https://cloud.redhat.com/blog/a-guide-to-openshift-and-uids](https://cloud.redhat.com/blog/a-guide-to-openshift-and-uids)

Especially:
```
The Container user is always a member of the root group, so it can read or write files accessible by GID=0. Any command invoked by the Entrypoint will be executed with this unprivileged UID and GID pair. That means, it is an unprivileged user executing the commands and the UID that will be used during execution is not known in advance. From the technical design perspective, that means, directories and files that may be written to by processes in the Container should be owned by the root group and be read/writable by GID=0. Files to be executed should also have group execute permissions. 
```

Reply:
and I guess this will need to be solved sooner rather than later, cause the jupyter lab are using a Jovyan user. not sure how easy that one will be to convert.

Reply:
computational services are definitely a bit easier on that side.